package-system/zlib/Findzlib.cmake

```diff
@@ -15,30 +15,21 @@ endif()
 set(zlib_INCLUDE_DIR ${CMAKE_CURRENT_LIST_DIR}/zlib/include)
 set(zlib_LIBS_DIR ${CMAKE_CURRENT_LIST_DIR}/zlib/lib)
 
-if (${PAL_PLATFORM_NAME} STREQUAL "Linux")
-    set(zlib_LIBRARY_DEBUG   ${zlib_LIBS_DIR}/libz.a)
-    set(zlib_LIBRARY_RELEASE ${zlib_LIBS_DIR}/libz.a)
-elseif(${PAL_PLATFORM_NAME} STREQUAL "Android")
-    set(zlib_LIBRARY_DEBUG   ${zlib_LIBS_DIR}/libz.a)
-    set(zlib_LIBRARY_RELEASE ${zlib_LIBS_DIR}/libz.a)
-elseif(${PAL_PLATFORM_NAME} STREQUAL "Windows")
+if(${PAL_PLATFORM_NAME} STREQUAL "Windows")
     set(zlib_LIBRARY_DEBUG   ${zlib_LIBS_DIR}/zlibstaticd.lib)
     set(zlib_LIBRARY_RELEASE ${zlib_LIBS_DIR}/zlibstatic.lib)
-elseif(${PAL_PLATFORM_NAME} STREQUAL "Mac")
-    set(zlib_LIBRARY_DEBUG   ${zlib_LIBS_DIR}/libz.a)
-    set(zlib_LIBRARY_RELEASE ${zlib_LIBS_DIR}/libz.a)
-elseif(${PAL_PLATFORM_NAME} STREQUAL "iOS")
+else()
     set(zlib_LIBRARY_DEBUG   ${zlib_LIBS_DIR}/libz.a)
     set(zlib_LIBRARY_RELEASE ${zlib_LIBS_DIR}/libz.a)
```

Comment:
is it intentional that both debug and release are the same library?, if so, there is no need to pass them in debug, you can just pass the `IMPORTED_LOCATION` one. You can have something like:
```
set_target_properties(${TARGET_WITH_NAMESPACE}  
    PROPERTIES
        IMPORTED_LOCATION       "${zlib_LIBRARY_RELEASE}"
)
if(zlib_LIBRARY_DEBUG)
    set_target_properties(${TARGET_WITH_NAMESPACE}  
        PROPERTIES
            IMPORTED_LOCATION_DEBUG "${zlib_LIBRARY_DEBUG}"
    )
endif()
```

Reply:
yes, I'll update this